### PR TITLE
help i can hold all these new breakdowns

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Adds in 4 new good breakdowns
Adds in 5 new bad breakdowns

### CODE REVIEW

File: code/modules/sanity/breakdowns.dm [hunk 1]
```diff
@@ -139,6 +128,106 @@
...
+	..()
+
+/datum/breakdown/positive/speedy_attacks
+	name = "Faster Basher"
+	duration = 15 MINUTES
+	restore_sanity_pre = 100
+
+	start_messages = list(
+		"Your hands feel faster to swing!",
+		"You sence your hands ready for combat!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:166]:
```suggestion
		"You steady your hands for combat!",
```

File: code/modules/sanity/breakdowns.dm [hunk 2]
```diff
@@ -139,6 +128,106 @@
...
+
+/datum/breakdown/positive/full_stomic
+	name = "Hungerless"
+	duration = 0
+	restore_sanity_pre = 100
+
+	start_messages = list(
+		"Your stomach feels full.",
+		"A knot in your stomach makes you feel not that hungery!",
+		"Weather it's the sight before you, or the acclamations of today you don't feel hungary.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:189]:
```suggestion
		"Whether it's your surroundings distracting you, or a matter of focus, hunger is all but gone from you.",
```

File: code/modules/sanity/breakdowns.dm [hunk 3]
```diff
@@ -139,6 +128,106 @@
...
+	..()
+
+/datum/breakdown/positive/full_stomic
+	name = "Hungerless"
+	duration = 0
+	restore_sanity_pre = 100
+
+	start_messages = list(
+		"Your stomach feels full.",
+		"A knot in your stomach makes you feel not that hungery!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:188]:
```suggestion
		"Your stomach no longer growls, your hunger recedes!",
```
Please for the love of god do **NOT** use **that** word here

File: code/modules/sanity/breakdowns.dm [hunk 4]
```diff
@@ -139,6 +128,106 @@
...
+	return TRUE
+
+
+/datum/breakdown/positive/primatic_fix
+	name = "Thinking Through"
+	duration = 15 MINUTES
+	restore_sanity_pre = 100
+
+	start_messages = list(
+		"Your mind clears letting you think of situations solution!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:211]:
```suggestion
		"Your mind clears, letting you think of solutions to your current predicaments!",
```

File: code/modules/sanity/breakdowns.dm [hunk 5]
```diff
@@ -139,6 +128,106 @@
...
+		"If you just think more you can make it through any problem",
+		"The mind can slove any problem thrown at it!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:213]:
```suggestion
		"The mind can solve any problem thrown at it!",
```

File: code/modules/sanity/breakdowns.dm [hunk 6]
```diff
@@ -139,6 +128,106 @@
...
+	)
+	end_messages = list(
+		"The world floods back losing any brute force thinking.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:216]:
```suggestion
		"Your thoughts acclimate, calming down. Your sudden rush of inspiration finally fades.",
```

File: code/modules/sanity/breakdowns.dm [hunk 7]
```diff
@@ -343,6 +428,132 @@
 	holder.max_level = max(holder.max_level - 20, 0)
 	..()
 
+/datum/breakdown/negative/cogintive_shutdown
+	name = "Mind-Whited"
+	duration = 15 MINUTES
+	restore_sanity_post = 50
+
+	start_messages = list(
+		"Your head fills will rushing throughs, all useless!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:437]:
```suggestion
		"Your head fills with rushing throughs, all useless!",
```

File: code/modules/sanity/breakdowns.dm [hunk 8]
```diff
@@ -343,6 +428,132 @@
...
+		"A loud echo fills your head losing any line of thought.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:438]:
```suggestion
		"A loud echo fills your head, making you lose any line of thought you had.",
```

File: code/modules/sanity/breakdowns.dm [hunk 9]
```diff
@@ -343,6 +428,132 @@
...
+		"It's hard to consitrate with all the white noise of the situations around you!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:439]:
```suggestion
		"It's hard to focus with all this white noise! I NEED SILENCE!",
```

File: code/modules/sanity/breakdowns.dm [hunk 10]
```diff
@@ -343,6 +428,132 @@
...
+	)
+	end_messages = list(
+		"The calmness comes back to your head.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:442]:
```suggestion
		"Your thoughts finally calm down, the noise in your head recedes.",
```

File: code/modules/sanity/breakdowns.dm [hunk 11]
```diff
@@ -343,6 +428,132 @@
...
+	holder.owner.stats.changeStat(STAT_COG, 15)
+	..()
+
+/datum/breakdown/negative/cold_shock
+	name = "Cold Shock"
+	duration = 0
+	restore_sanity_post = 50
+
+	start_messages = list(
+		"The worlds heat leaves you!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:463]:
```suggestion
		"You suddenly feel heat leave your body, hands freezing, body shivering! It's cold... so cold...",
```

File: code/modules/sanity/breakdowns.dm [hunk 12]
```diff
@@ -343,6 +428,132 @@
...
+		"The chill of the cold reality hits you like a sheet of ice.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:464]:
```suggestion
		"You feel a horrible chill down your spine...", // Terraria reference
```

File: code/modules/sanity/breakdowns.dm [hunk 13]
```diff
@@ -343,6 +428,132 @@
...
+		"Frost comes to the mind, leaving the body shivering!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:465]:
```suggestion
		"Your body shivers uncontrollably, has this place always been THIS cold?!",
```

File: code/modules/sanity/breakdowns.dm [hunk 14]
```diff
@@ -343,6 +428,132 @@
...
+	)
+	end_messages = list(
+		"The warm from other sources is required to survive.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:468]:
```suggestion
		"Maybe all this cold was just psychosomatic...?",
```

File: code/modules/sanity/breakdowns.dm [hunk 15]
```diff
@@ -343,6 +428,132 @@
...
+	..()
+
+/datum/breakdown/negative/unsightly
+	name = "Sickening Sight"
+	duration = 0
+	restore_sanity_post = 50
+
+	start_messages = list(
+		"The smells, the sights, its all to much for your weak stomach!",
+		"The reatched sights and smells leave you vomiting outwards.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:482]:
```suggestion
		"The wretched sights and smells leave you vomiting outwards.",
```

File: code/modules/sanity/breakdowns.dm [hunk 16]
```diff
@@ -343,6 +428,132 @@
...
+		"The enviroment is to unpleasent for you.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:483]:
```suggestion
		"This is too much to handle! I'm feeling sick...!",
```

File: code/modules/sanity/breakdowns.dm [hunk 17]
```diff
@@ -343,6 +428,132 @@
...
+		return FALSE
+	return TRUE
+
+/datum/breakdown/negative/fluid_fear
+	name = "Fluid Fear"
+	duration = 15 MINUTES
+	restore_sanity_pre = 50
+
+	start_messages = list(
+		"Your arms feel weak, your legs shakey!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:504]:
```suggestion
		"Your arms feel heavy, your knees are weak!",
```

File: code/modules/sanity/breakdowns.dm [hunk 18]
```diff
@@ -343,6 +428,132 @@
...
+		"You feel like a wet noodle!",
+		"Your body feels like liquid and sways!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:506]:
```suggestion
		"You feel like you can barely throw a punch around!",
```

File: code/modules/sanity/breakdowns.dm [hunk 19]
```diff
@@ -343,6 +428,132 @@
...
+	)
+	end_messages = list(
+		"You feel more soild again and can fight a bit better.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:509]:
```suggestion
		"Your fighting instincts are back to you, you feel embarrassed at your lack of control.",
```

File: code/modules/sanity/breakdowns.dm [hunk 20]
```diff
@@ -343,6 +428,132 @@
...
+#define GASPING_BREATHS_COOLDOWN rand(5 SECONDS, 15 SECONDS)
+
+/datum/breakdown/negative/gasping_breaths
+	name = "Gasping Breaths"
+	duration = 5 MINUTES
+	restore_sanity_post = 50
+	var/pick_time = 0
+
+	start_messages = list(
+		"The walls close in your mouth breaths heavily, you must keep your mouth free.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:531]:
```suggestion
		"The walls close in, your mouth breaths heavily - everything around you seems bent on not letting you breathe, even what you wear...",
```

File: code/modules/sanity/breakdowns.dm [hunk 21]
```diff
@@ -343,6 +428,132 @@
...
+		"Breathing is so hard, you must keep your mouth free from any objects blocking vital air!",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:532]:
```suggestion
		"Your throat closes, your words are raspy, this thing in your mouth doesn't let you breathe, take it out!",
```

File: code/modules/sanity/breakdowns.dm [hunk 22]
```diff
@@ -343,6 +428,132 @@
...
+		"The thin air makes anything obstructing your mouth feel like you will faint.",
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:533]:
```suggestion
		"This thing in your face is blocking your breathable air! Take it out! TAKE IT OUT! You must breathe!!",
```

File: code/modules/sanity/breakdowns.dm [hunk 23]
```diff
@@ -343,6 +428,132 @@
...
+	)
+	end_messages = list(
+		"The air is thick again and you can give breath with easy."
```
Comment (ThePainkiller) [inline code/modules/sanity/breakdowns.dm:536]:
```suggestion
		"Your hypochondria fades away, your breaths steady themselves as you calm down."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Trilbyfication fixies

**Comment by ThePainkiller** [comment]
While we're at it, can we please remove the hysteric breakdown that causes you to sob and scream on the floor? It's handled terribly ICly and a bit awkward to properly roleplay that one out or even explain it

File: code/modules/sanity/breakdowns.dm (hunk 19 above)
Comment (Trilbyspaceclone) [reply code/modules/sanity/breakdowns.dm:509]: Emotional feeeeling is trying to be voided atm

File: code/modules/sanity/breakdowns.dm (hunk 21 above)
Comment (Trilbyspaceclone) [reply code/modules/sanity/breakdowns.dm:532]: These are generlized as they work with or without anything on the mask slot

File: code/modules/sanity/breakdowns.dm (hunk 19 above)
Comment (ThePainkiller) [reply code/modules/sanity/breakdowns.dm:509]: Oke, editing so that it's out

File: code/modules/sanity/breakdowns.dm (hunk 22 above)
Comment (Trilbyspaceclone) [reply code/modules/sanity/breakdowns.dm:533]: What if theirs nothing on your face?

File: code/modules/sanity/breakdowns.dm (hunk 21 above)
Comment (ThePainkiller) [reply code/modules/sanity/breakdowns.dm:532]: Alright, fair!

File: code/modules/sanity/breakdowns.dm (hunk 22 above)
Comment (ThePainkiller) [reply code/modules/sanity/breakdowns.dm:533]: Good point, nevermind this one

**Pull request merged**